### ISSUE TITLE
Hand switching

### ISSUE THREAD

**Issue by Llamalland** [ForFunLabs/ElevenPublicBugsAndAsks#194]
Like Timo Boll is famous for, we should be able to switch hands. https://www.youtube.com/watch?v=239JKqz4YfY

Totally ok (and probably appropriate if the way it is done is cumbersome and difficult, as it is difficult in real life. Maybe hold a button on your non-bat controller and bring your controllers close together, and similar thing to switch back? Must be opt-in, in my opinion, don't want anyone switching hands by accident.

**Comment by Mockingblrd** [comment]
I agree with Llammalland; that potential feature is the one I'd love the most in a future update. The next best potential, future feature would be if both hands show up virtually instead of just one hand where the other hand shows up as a controller. I think this is possible because the first steps app on the oculus quest during the tutorial demonstrates how both hands show up when interacting with the ping pong and paddle, and it works in a way where ambidextrous people can switch paddles easily at any given time.

**Comment by carrotstien** [comment]
> I think this is possible because the first steps app on the oculus quest during the tutorial demonstrates how both hands show up when interacting with the ping pong and paddle

are you able to hold the paddle anywhere you want in this example...or does it snap to a specific position...and hand pose?

**Comment by Mockingblrd** [comment]
I'm able to hold the paddle anywhere I want in the first steps game on the oculus quest, and both hands show up as virtual hands instead of one virtual hand and one virtual controller.
